packages/studio-plugin/src/orchestrators/ParsingOrchestrator.ts

```diff
@@ -208,6 +216,14 @@ export default class ParsingOrchestrator {
     );
   };
 
+  private updateFilepathToDependencyTree(absPath: string) {
```

Comment:
do we need to think about calling this function as result of HMR triggered by a change to a dependency of a component file?

Reply:
decided to hold off on this until we support general HMR for this case, not just CSS!